### PR TITLE
qbittorrent: update dependencies

### PR DESCRIPTION
**Pull request by LABPBS**

### CODE REVIEW

File: mingw-w64-qbittorrent/PKGBUILD [hunk 1]
```diff
@@ -4,18 +4,21 @@ _realname=qbittorrent
...
 depends=("${MINGW_PACKAGE_PREFIX}-boost"
-         "${MINGW_PACKAGE_PREFIX}-qt5"
+         "${MINGW_PACKAGE_PREFIX}-qt5-base"
+         "${MINGW_PACKAGE_PREFIX}-qt5-winextras"
          "${MINGW_PACKAGE_PREFIX}-libtorrent-rasterbar"
          "${MINGW_PACKAGE_PREFIX}-zlib")
 optdepends=("${MINGW_PACKAGE_PREFIX}-python: needed for torrent search tab")
-makedepends=("${MINGW_PACKAGE_PREFIX}-pkg-config")
+makedepends=("${MINGW_PACKAGE_PREFIX}-pkg-config"
+             "${MINGW_PACKAGE_PREFIX}-qt5-svg"
```
Comment (lazka) [inline mingw-w64-qbittorrent/PKGBUILD:20]: why is qt5-svg not in depends?

Comment (LABPBS) [reply mingw-w64-qbittorrent/PKGBUILD:20]: I put it in depends at first, but then i thought id check the arch package and its a makedepend there, then i thought, hey lets check ntldd
```
ntldd /clang64/bin/qbittorrent.exe | grep -vi 'C:\\Windows\\'
zlib1.dll
libtorrent-rasterbar.dll
libcrypto-1_1-x64.dll
Qt5Widgets.dll
Qt5WinExtras.dll
Qt5Gui.dll
Qt5Network.dll
Qt5Xml.dll
Qt5Core.dll
libc++.dll
libunwind.dll
```
and i thought look it doesnt depend on Qt5Svg

Comment (lazka) [reply mingw-w64-qbittorrent/PKGBUILD:20]: then why is it in makedepends if it isn't used?

Comment (LABPBS) [reply mingw-w64-qbittorrent/PKGBUILD:20]: https://github.com/qbittorrent/qBittorrent/blob/release-4.3.7/src/CMakeLists.txt#L63
the cmakelists requires it

Comment (LABPBS) [reply mingw-w64-qbittorrent/PKGBUILD:20]: Hold on, i just realized this is built with configure 🤦

Comment (lazka) [reply mingw-w64-qbittorrent/PKGBUILD:20]: but I see `plugins/imageformats` in qt5-svg. couldn't it be that svg support is dynamically loaded via plugins at runtime?

It's that way in gtk at least..

Comment (LABPBS) [reply mingw-w64-qbittorrent/PKGBUILD:20]: https://github.com/qbittorrent/qBittorrent/blob/release-4.3.7/configure.ac#L143
i know nothing about autotools but maybe it wont fail without Qt5Svg

Comment (lazka) [reply mingw-w64-qbittorrent/PKGBUILD:20]: yes, svg support is dynamically loaded, please add it back to depends.

**Pull request merged**